TrustModel.md

```diff
@@ -0,0 +1,426 @@
+trusts the code provided by the workload provider to operate across their data, however they do 
+not trust the Orchestration Operator or Cloud Operator with their data and wish to ensure 
+data confidentiality.
+
+The workload provider must not trust the Orchestration Operator or Cloud Operator  
+otherwise they risk loosing the trust of the data owner. This means the workload provider must take 
+steps to ensure the integrity of the code and that the data is appropriately protected, they may also 
+wish to maintain confidentiality of their code from other personas.
+
+
+## Lifecycle 
+The Cloud Native security Paper [^2] describes the lifecycle relating to workloads in four phases 
+- Develop
+- Distribute
+- Deploy
+- Runtime
+
+When we consider confidential containers there are lifecycle phases required before arriving at 
+the runtime environment in which the workload is truely confidential. Threats can exist during 
+these lifecycle phases and trust must be considered in the context of the lifecycle of the 
+environment supporting confidential containers.
+
+
+![Lifecycle Phases](./images/LifecyclePhases.svg)
+
+Taking the four phases above we can map this as :
+
+#### Develop
+ - Preparation and build of container images
+ - Preparation and build of the trusted boot image for the TEE. 
+ - Signatures and encryption keys for containers are prepared and available
+ 
+````` Completing building the boot image triggers the next phase.`````
+
+#### Distribute
+The Distribute stage includes all preparations before the guest executes its first instructions, 
+and it will run some general and TEE-specific operations.
+- configuring the cluster with TEE
+- making the boot image available
+- taking initial measurements
+- provision secrets.
+- allocation of guest resource
+- loading guest fw and configuration occurs 
+- TEE-specific steps relating to preparing attestation/secrets etc
+
+The involved components include the ones that directly associated with the
+protections that TEEs provide (e.g, tdx seam-loader, tdx module, psp/sev firmware and so on),
+Pre-OS (e.g, guest firmware, guest kernel, initrd and so on) and specific components (e.g,
+kata-agent, attestation-agent and so on). Also includes preparing secrets for delivery to the TEE.
+
+```The first instruction being executed within the trusted boot image happens in the next phase```
+
+#### Deploy
+  - pod sandbox preparation
+  - "pre flight checks" include attestation measurment checks
+  - secret provisioning linked to boot attestation
+  
+  ```Receiving the request to pull and deploy a container happens in the runtime phase```
+
+
+#### Runtime
+- post boot attestation and secret provisioning
+- procedure of image pulling, unpacking and 
+  launching.
+- the workload in the container starts to  run in TEEs.
+
+
+These execution phases are sufficient to describe the execution of any forms of confidential
+container. 
```

Comment:
It might be handy to give an example of what commands or functions are running in each phase. Distribution vs Deployment is a little fuzzy to me.